.github/workflows/conformance.yml

```diff
@@ -0,0 +1,292 @@
+name: Talos Conformance
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+  schedule:
+    # Run weekly.
+    - cron: '* * * * 1'
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  setup-and-test:
+    runs-on: ubuntu-22.04
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        include:
+          # --- Cilium v1.15 ---
+
+          - name: 'v1.15'
+            cilium-version: "v1.15.1"
+            talos-version: 'v1.6.2'
```

Comment:
How about testing multiple Talos versions? E.g., the two latest minor versions.

Reply:
Talos mainly only supports the latest minor release + critical fixes for n-1: https://www.talos.dev/v1.6/introduction/support-matrix/

Reply:
Makes sense, I'm thinking we're going to want to maybe have two sets of tests... a PR that's just most recent and a weekly one which will do a full matrix of:

(n...n-2 cilium versions) X (n...n-1 talos versions) X (config tuples) = 3 * 2 * 6 = 36

Which is probably too heavy for a PR workflow.